src/liquid/components/ld-sidenav/ld-sidenav.tsx

```diff
@@ -389,7 +406,7 @@ export class LdSidenav {
   }
 
   componentDidLoad() {
-    this.ldSidenavCollapsedChange.emit(this.collapsed)
+    this.ldSidenavCollapsedChange.emit(this.collapsible && this.collapsed)
```

Comment:
Shouldn't the whole event not be emitted when `this.collapsible === false`?

Reply:
If I change this as proposed everything still works, but then it's kind of inconsistent with the other events being emitted on componentDidLoad. I see this more as a status update: The component emits "hey, I'm not collapsed, open and closable" when it's loaded. I think that's fine.